.travis.yml

```diff
@@ -1,17 +1,40 @@
 ---
+# https://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/
+sudo: false
+language: ruby
+dist: trusty
```

Comment:
Looking through the tickets on Rubinous, they're no longer supporting 2.X or Precise:

> Even if we were to support Rubinius 2.x, we are not able to build binaries for current versions of Rubinius on Precise because the LLVM packages are too old. We also cannot use the custom LLVM packages because other system libraries are too old. This issue was causing significant confusion for people trying to test with Rubinius on Travis.

> Precise end-of-life is April 2017, a few short months away. Other providers, like Heroku, completed migrations from Precise years ago. With our very limited resources we have no extra time to waste supporting OS distros and versions that are easily replaced with readily available modern versions.